### PR TITLE
[CI] [GHA] Drop Python 3.8 testing in GHA

### PR DESCRIPTION
**Pull request by akashchi**
### Tickets:
 - *151796*

### CODE REVIEW

File: .github/dockerfiles/ov_test/fedora_33/Dockerfile
Comment (akashchi) [inline .github/dockerfiles/ov_test/fedora_33/Dockerfile:1]: We have to update to Fedora 33 as it has Python 3.9 as default.

File: .github/workflows/job_build_linux.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ on:
       arch:
         description: 'Target architecture'
         type: string
-        default: 'x86_64'
+        default: 'intel64'
```
Comment (ilya-lavrenov) [inline .github/workflows/job_build_linux.yml:28]: why do you use intel64 instead of x86_64 ? we typically use x86_64  in OpenVINO

Comment (akashchi) [reply .github/workflows/job_build_linux.yml:28]: I wanted to align it with artifacts storing action: https://github.com/openvinotoolkit/openvino/blob/c9288cce97bcf5e6337449768ec0c71492ce0621/.github/workflows/job_build_linux.yml#L124
@akladiev, can we use x86_64 for the `target_arch` input? Granted we updated the enum in https://github.com/openvinotoolkit/openvino/blob/c9288cce97bcf5e6337449768ec0c71492ce0621/.github/actions/common/constants.py#L9

Comment (akladiev) [reply .github/workflows/job_build_linux.yml:28]: It's target arch from this path in OV artifacts:
image
In internal validation this value is used at least in fuzzing tests where you need to provide a full path to libopenvino.so to test (input for-object option for llvm-cov tool), maybe also somewhere else

Comment (ilya-lavrenov) [reply .github/workflows/job_build_linux.yml:28]: intel64 is used within package
x86_64 is used as package name, arch names

File: .github/workflows/linux_arm64.yml [hunk 2]
```diff
@@ -180,147 +98,256 @@ jobs:
...
-          artifacts: |
-            ${{ env.BUILD_DIR }}/openvino_package.tar.gz
-            ${{ env.BUILD_DIR }}/openvino_developer_package.tar.gz
-            ${{ env.BUILD_DIR }}/openvino_tests.tar.gz
-            ${{ env.MANIFEST_PATH }}
-          storage_dir: ${{ env.PRODUCT_TYPE }}
-          storage_root: ${{ env.ARTIFACTS_SHARE }}
+            -DOV_CPU_AARCH64_USE_MULTI_ISA=ON
+
+#  Build:
```
Comment (ilya-lavrenov) [inline .github/workflows/linux_arm64.yml:103]: should we drop this commented piece of code?

File: .github/workflows/linux_arm64.yml [hunk 3]
```diff
@@ -180,147 +98,256 @@ jobs:
...
-        uses: ./openvino/.github/actions/store_artifacts
-        with:
-          artifacts: |
-            ${{ env.BUILD_DIR }}/openvino_package.tar.gz
-            ${{ env.BUILD_DIR }}/openvino_developer_package.tar.gz
-            ${{ env.BUILD_DIR }}/openvino_tests.tar.gz
-            ${{ env.MANIFEST_PATH }}
-          storage_dir: ${{ env.PRODUCT_TYPE }}
-          storage_root: ${{ env.ARTIFACTS_SHARE }}
+            -DOV_CPU_AARCH64_USE_MULTI_ISA=ON
```
Comment (ilya-lavrenov) [inline .github/workflows/linux_arm64.yml:101]: It's enabled by default, we can drop it.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**